src/org/org.ts

```diff
@@ -1506,13 +1516,17 @@ export class Org extends AsyncOptionalCreatable<Org.Options> {
    * @param where clause to query for
    * @private
    */
-  private async querySandboxProcess(where: string): Promise<SandboxProcessObject> {
+  private async querySandboxProcess(where: string, refresh = false): Promise<SandboxProcessObject> {
     const soql = `SELECT ${sandboxProcessFields.join(
       ','
     )} FROM SandboxProcess WHERE ${where} ORDER BY CreatedDate DESC`;
-    const result = (await this.connection.tooling.query<SandboxProcessObject>(soql)).records.filter(
-      (item) => !item.Status.startsWith('Del')
-    );
+    const result = (await this.connection.tooling.query<SandboxProcessObject>(soql)).records.filter((item) => {
+      if (refresh) {
+        return !item.Status.startsWith('Completed');
```

Comment:
this is probably bad (resuming a finished refresh process would return no records, fix later!